src/main/resources/stubs/time/sleep.gobra

```diff
@@ -0,0 +1,13 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in https://golang.org/LICENSE
+
+// Signatures for the public declarations in file
+// https://github.com/golang/go/blob/master/src/time/sleep.go
+
+package time
+
+// Sleep pauses the current goroutine for at least the duration d.
+// A negative or zero duration causes Sleep to return immediately.
+decreases
+func Sleep(d Duration)
```

Comment:
The documentation does not clearly state that `Sleep` terminates always. In particular, it says that it "returns immediately" when the argument is less than or equal to 0 and it waits at least `d` when it is positive - it does not specify that it terminates in this case. I would change the termination measure to `decreases _ if d <= 0` (or something morally equivalent).

Reply:
Ok, I assumed that it will always terminate in finite time. But I change it, such that we are on the safe side